docker/latest/README.md

```diff
@@ -1,10 +1,10 @@
-### Build fledge image
+### Build fledge Ubuntu 18.04 image
 
 ```
-    $ docker build --no-cache --tag fledge:latest .
+    $ docker build --no-cache --tag fledge:latest -f Dockerfile.ubuntu1804 .
```

Comment:
We better leave default as is for Ubuntu 18.04; we can say in doc, in case you need different base OS, you can choose different dockerfiles e.g. -f Dockerfile.ubuntu2004, -f Dockerfile.ubuntu2004-aarch64 (armv8/arm64 whatever), -f Dockerfile.centos7 etc..

Reply:
I think even for Ubuntu 18.04 we keep Dockerfile.ubuntu1804 as the file would get easily differentiated. Also in CI, we have used the -f parameter in the docker build command.